### PR TITLE
[4.13] GCP IPI XPN

### PR DESCRIPTION
**Pull request by jianli-wei**
Cont. to https://github.com/openshift/release/pull/35800

### CODE REVIEW

**Comment by jianli-wei** [comment]
@jianlinliu please help review, thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -1661,7 +1661,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-workers-rhel8
-- as: gcp-ipi-xpn-techpreview-p2-f7
+- as: gcp-ipi-xpn-p2-f7
   cron: 18 3 30,7,14,21 * *
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1665]: Need to run https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh to generate a new cron entry

**Comment by jianlinliu** [comment]
cc @liangxia and @jhou1 for review and approve

**Pull request merged**